app/src/pages/NotFoundPage.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function NotFoundPage() {
+  return (
+    <p>
+      <h1>404</h1>
```

Comment:
I don't think 404 is particularly useful for users.

This site has a great example 404 page: https://covidtracking.com/thispagedoesntexist

No need to do something that fancy, but it's a cool idea for inspiration to link them to other parts. Anyway for now I'd just drop the 404 and stick with page not found

Reply:
Sure! I actually hadn't meant for any of this text to stay :)
I'll update to just Page Not Found for now though!